Tools/CISupport/ews-build/steps.py

```diff
@@ -5646,6 +5666,12 @@ def __init__(self, rebase_enabled=True, **kwargs):
         self.rebase_enabled = rebase_enabled
         self.contributors = {}
 
+    def commit_count(self):
```

Comment:
is it actually the count of commits or number of commits to canonicalize? If so, better to name it accordingly to avoid confusion.

Reply:
Is suppose yes, this is actually the count of the number of commits to canonicalize, which is based on the number of commits in the pull-request.